client/src/AdvancedSearch.elm

```diff
@@ -136,35 +148,62 @@ view routingMsg formMsg data categories =
-    in
-    [ h1 [] [ text "Advanced Search" ]
-    , hr [] []
-    , form [ onSubmit << routingMsg <| SearchResults data Pagination.default, class "advanced-search" ]
-        [ div [ class "form-group" ]
-            [ legend [ class "font-weight-bold", for "keywords" ] [ text "Keywords: " ]
-            , input
-                [ id "keywords"
-                , class "form-control"
-                , type_ "search"
-                , value data.query
-                , onInput <| formMsg << KeywordInput
-                , Aria.label "Search Terms"
+
+        submitButton =
+            button [ class "shrink-0", type_ "submit" ] [ searchSvg ]
+
+        searchBar : Html msg
+        searchBar =
+            div [ class "tw:w-full tw:py-[16px] tw:px-[18px] bg-white tw:rounded-[16px] tw:border tw:border-[rgba(232,231,230,1)] tw:flex tw:gap-[8px] tw:items-center " ]
+                [ submitButton
+                , input
+                    [ id "keywords"
+                    , class "tw:block tw:w-full tw:placeholder:text-[rgba(187,182,179,1)]"
+                    , type_ "search"
+                    , value data.query
+                    , onInput <| formMsg << KeywordInput
+                    , Aria.label "Search Terms"
+                    , placeholder "Search"
+                    ]
+                    []
+                , categorySelect
                 ]
+
+        searchIn =
+            if showDetail then
+                [ div []
+                    [ label [ class "mr-4 font-weight-bold" ] [ text "Search In: " ]
+                    , radioInput SearchTitles .searchIn Search.Titles "Titles"
+                    , radioInput SearchTitlesAndDescriptions
+                        .searchIn
+                        Search.TitlesAndDescriptions
+                        "Titles & Descriptions"
+                    ]
+                ]
+
+            else
                 []
-            ]
-        , div []
-            [ label [ class "mr-4 font-weight-bold" ] [ text "Search In: " ]
-            , radioInput SearchTitles .searchIn Search.Titles "Titles"
-            , radioInput SearchTitlesAndDescriptions
-                .searchIn
-                Search.TitlesAndDescriptions
-                "Titles & Descriptions"
-            ]
-        , legend [ class "font-weight-bold" ] [ text "Filters:" ]
-        , div [ class "row align-items-center" ]
-            [ div [ class "col filters mb-3" ] filterCheckboxes
-            , div [ class "col-auto mb-3" ] [ categorySelect ]
-            ]
-        , button [ class "mb-3 btn btn-primary", type_ "submit" ] [ text "Submit" ]
+    in
+    (if showDetail then
+        [ h1 [] [ text "Advanced Search" ]
+        , hr [ class "tw:pb-[36px]" ] []
         ]
-    ]
+
+     else
+        []
+    )
+        ++ form [ class "tw:pb-[28px] tw:flex tw:flex-col tw:gap-[16px]", onSubmit << routingMsg <| SearchResults data Pagination.default, class "advanced-search" ]
+            [ searchBar
```

Comment:
From my understanding, the "Advanced search" is still using keywords for searching, but currently it's unclear that the search form expects keywords to be provided
<img width="935" height="284" alt="image" src="https://github.com/user-attachments/assets/ec6b5198-6bec-4dc0-9b80-2683539575a7" />

The previous version:
<img width="977" height="373" alt="image" src="https://github.com/user-attachments/assets/e50c3d35-8fbe-4f3e-b2c8-b38974985bdb" />

Does the new design ditch keyword search altogether?

Reply:
I'll take a look if the backend still uses keywords at all

Reply:
Ah, okay, the backend looks through `(name, sku, keywords)` or `(name, sku, long description, keywords)` depending on "Search In" option, so I guess "Keywords" indeed could be ommitted